api/v2alpha1/go/pipelinespec/pipeline_spec.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.33.0
-// 	protoc        v3.17.3
+// 	protoc-gen-go v1.35.1
```

Comment:
what version of protoc do you have installed? I think we'll need to pick one that we all standardize on. I've been using protoc-21.12.